### PR TITLE
scenario added for testing special invalid and valid username

### PR DESCRIPTION
**Pull request by jaggu4444**
A new scenario has been added so that it can check for the valid and invalid users with some special characters.

### CODE REVIEW

File: tests/ui/features/other/login.feature [hunk 1]
```diff
@@ -22,6 +22,23 @@ So that unauthorised access is impossible
 		And I logout
 		And I login with username "guiusr1" and password "pwd"
 		Then I should be redirected to a page with the title "Files - ownCloud"
+		
+	Scenario: use the webUI to create a user with special valid characters
+		Given a regular user exists
```
Comment (individual-it) [inline tests/ui/features/other/login.feature]: why do you need the regular user here?

File: tests/ui/features/other/login.feature [hunk 2]
```diff
@@ -22,6 +22,23 @@ So that unauthorised access is impossible
...
+		And I am on the users page
+		When I create a user with the name "@-_.'" and the password "pwd"
+		And I logout
+		And I login with username "@-_.'" and password "pwd"
+		Then I should be redirected to a page with the title "Files - ownCloud"
+		
+	Scenario: use the webUI to create a user with special invalid characters
+		Given I am logged in as admin
+		And I am on the users page
+		When I create a user with the name "as#" and the password "pwd"
```
Comment (individual-it) [inline tests/ui/features/other/login.feature]: please test some more different char like `/&'?`

File: tests/ui/features/other/login.feature [hunk 3]
```diff
@@ -13,15 +13,37 @@ So that unauthorised access is impossible
 		And I am on the login page
 		When I login as a regular user with a correct password
 		Then I should be redirected to a page with the title "Files - ownCloud"
-	
-	Scenario: use the webUI to create a simple user
-		Given a regular user exists
-		And I am logged in as admin
+		
+		Scenario: use the webUI to create a simple user
```
Comment (shresthamoheet) [inline tests/ui/features/other/login.feature]: wrong indent

**Comment by individual-it** [comment]
moved PR to owncloud origin https://github.com/owncloud/core/pull/29931

**Pull request closed without merging**